test/transferSpec.js

```diff
@@ -68,16 +68,14 @@ describe('Transfer methods', function () {
   describe('sendTransfer', function () {
     it('submits a transfer', function * () {
       nock('http://red.example')
-        .put('/transfers/6851929f-5a91-4d02-b9f4-4ae6b7f1768c', this.ledgerTransfer)
-        .basicAuth({user: 'mike', pass: 'mike'})
+        .put('/transfers/6851929f-5a91-4d02-b9f4-4ae6b7f1768c?token=abc', this.ledgerTransfer)
```

Comment:
You can use `nock`'s `.query` (https://github.com/node-nock/nock#specifying-request-query-string)